src/SamplesApp/UITests.Shared/Windows_UI_Xaml_Controls/SymbolIconTests/SymbolIcon_Generic.xaml.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection.Emit;
+using Uno.UI.Samples.Controls;
+using Windows.UI.Xaml.Controls;
+
+namespace UITests.Windows_UI_Xaml_Controls.SymbolIconTests
+{
+	[Sample("SymbolIcon")]
+	public sealed partial class SymbolIcon_Generic : Page
+    {
+        public SymbolIcon_Generic()
+        {
+            this.InitializeComponent();
+        }
+
+		public List<SymbolListItem> Symbols { get; } =
```

Comment:
Codacy found an issue: [Refactor this property to use a generic collection designed for inheritance.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=6148979)

Reply:
N/A